xwiki.lua

```diff
@@ -54,10 +54,13 @@ end
 
 -- Table to store footnotes, so they can be included at the end.
 local notes = {}
+local bullets = {}
+local orders = {}
 
 -- Blocksep is used to separate block elements.
 function Blocksep()
-  return "\n\n"
+  return bullets[1] and "\n" or orders[1] and "\n" or "\n\n"
+--  return "\n\n"
```

Comment:
Just delete code, don't comment it out.  There are several other places where that can be done, I won't call them all out individually

Reply:
Done.